hardware/snap_check_psl

```diff
@@ -20,19 +20,24 @@
 ############################################################################
 #
 
-export PSL_DCP_TYPE="UNKNOWN"
-
+if [[ $# -eq 2 ]]; then
```

Comment:
'[' should be used only where necessary (e.g. to get regular expression support).